RequestHeaderSentValue.js

```diff
@@ -1,14 +1,25 @@
 var RequestHeaderSentValue = function() {
 	this.evaluate = function(context) {
 
-		var exchange = this.req.getLastExchange();
+		var exchange;
+
+		if(this.req !== null) {
+			exchange = this.req.getLastExchange();
+		} else {
+			this.req = context.getCurrentRequest();
+			exchange = this.req.getLastExchange();
```

Comment:
try not assigning the `this.req` value here. instead you can do in just one line:

`exchange = context.getCurrentRequest().getLastExchange();`